discovery-provider/src/queries/health_check.py

```diff
@@ -50,6 +50,7 @@ def _get_db_block_state(latest_blocknum, latest_blockhash):
         )
         health_results["block_difference"] = block_difference
         health_results["maximum_healthy_block_difference"] = default_healthy_block_diff
+        health_results.update(disc_prov_version)
```

Comment:
looks good to me, just to confirm this would result in the following?

```
{
  "service": "discovery-provider",
  "version": "0.3.5",
  "block_difference": 0,
  "db": {
    "blockhash": "0x23a3ec9d5b2e16370621fa06ac2d3370276ffe67592eaaf7ad1f27a0780dd69c",
    "is_current": true,
    "number": 14725698,
    "parenthash": "0x23b9b7668f735805bbfa6ddb3d4c4bdba49084106b4d8d3b8259d2ab73b1cd31"
  },
  "git": "5beef4f802b285bbd20f8cccd5c0a33407cd95d3",
  "maximum_healthy_block_difference": 100,
  "web": {
    "blockhash": "0x23a3ec9d5b2e16370621fa06ac2d3370276ffe67592eaaf7ad1f27a0780dd69c",
    "blocknumber": 14725698
  }
}
```

Reply:
yep, looks like that